### ISSUE TITLE
[BUG] Paulmann 501.34 Wall Switch Button 2 and 4 long presses do not work

### ISSUE THREAD

**Issue by itsmicash** [zigpy/zha-device-handlers#3382]
### Bug description

@redfusion83 @drueppela @MattFromGer @TheJulianJES 

coming from this thread: [Issue](https://github.com/zigpy/zha-device-handlers/issues/2662)

Hi guys,

I have issues with button A0 and B0 (or button 2 and 4)
all button presses work except long press and press and realease on these 2 buttons. hope someone can support me.
also tried on a second paulmann switch, same.

Here are my events in zha if i trigger them, they get recognized but i cannot build my automations on them.

Button 2 long press & release
![image](https://github.com/user-attachments/assets/51bd4e56-8089-44ff-a513-f41056f6c7d7)

Button 2 long press
![image](https://github.com/user-attachments/assets/425babdb-8c74-4320-b117-0fc4c3f48d7b)

Button 4 long press & release
![image](https://github.com/user-attachments/assets/be1c297f-9971-4525-87db-65b9274ed247)

Button 4 long press
![image](https://github.com/user-attachments/assets/586c8710-e0bf-41cd-b852-38c11b89f773)

I'm really struggling and don't know what to do. 

### Steps to reproduce

1. create automation
2. select switch as trigger
3. select button 2 or 4 continuously pressed
4. save and trigger e.g. button 2 long press, it wont get triggered

### Expected behavior

it should be possible to trigger it

### Screenshots/Video

see above

### Logs

get more details like device signature from here
[issue](https://github.com/zigpy/zha-device-handlers/issues/2662#issue-1953730625)

**Comment by thwgmx** [comment]
Same problem here. I also tried a blueprint with the adaptive lights add-on

**Comment by gexxx05** [comment]
Yeah, i have the same problem, but you can create a custom trigger as a workaround for this:

1. In Home Assistant go to Developer Tools and Events, listen for Event "zha_event" and make a long press on the off button you want. After the press, stop listening and save the two results. One has the command "move" (Long press) and one has the command "stop" (Realese after long press).

2. Copy all the content from data section and remove the unique_id part.
![2024-12-14 13_42_33-Entwicklerwerkzeuge – Home Assistant – Firefox Developer Edition](https://github.com/user-attachments/assets/f12825da-646a-4663-945c-eff4e3712f16)

3. In your automation create a custom trigger with event type "zha_event" and copy the data from above.
![2024-12-14 13_41_54-Einstellungen – Home Assistant – Firefox Developer Edition](https://github.com/user-attachments/assets/f9474aa3-db56-49b6-9176-a8a5a0277cf0)

4. Now you should be able to dimm your lights.

**Comment by itsmicash** [comment]
@gexxx05 
thanks for the hint. in the meanwhile i migrated to z2mqtt, which does the job much better for the paulmann switch and in general for all my other devices. I hope your suggestion helps others with that issue.

**Comment by voelkerb** [comment]
Bump! please correct in ZHA so that the workaround is not needed and blueprints can be used.
